app/controllers/admin/users_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Admin::UsersController < ApplicationController
   before_action :authorize_admin!
   before_action :authorize_super_admin!, only: [ :update, :destroy ]
+  after_action :remove_user_locks, only: [:destroy]
```

Comment:
user does not have has_many :locks -> when user is soft_deleted the lock
stays behind

On Tue, Mar 3, 2015 at 11:06 AM, Jon Moter <notifications@github.com> wrote:

> In app/controllers/admin/users_controller.rb
> <https://github.com/zendesk/samson/pull/257#discussion_r25715816>:
>
> > @@ -1,6 +1,7 @@
> >  class Admin::UsersController < ApplicationController
> >    before_action :authorize_admin!
> >    before_action :authorize_super_admin!, only: [ :update, :destroy ]
> > +  after_action :remove_user_locks, only: [:destroy]
>
> In that case, curious that we have some soft-deleted users that own locks
> that haven't been deleted.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/257/files#r25715816>.
>